tests/unit/test_osti_single.py

```diff
@@ -0,0 +1,173 @@
+    assert parsed_record.abstract == expected
+
+
+def test_authors(parsed_record):
+    """Test extracting authors"""
+    expected = [{'full_name': 'Bjorken, J.D.',
+                 'raw_affiliations': [
+                     {'value': 'Stanford Univ., Stanford, CA (United States)',
+                      'source': 'OSTI'}]},
+                {'full_name': 'Cline, D.',
+                 'raw_affiliations': [
+                     {'value': 'Univ. of Wisconsin, Madison, WI (United States)',
+                      'source': 'OSTI'}]},
+                {'full_name': 'Mann, A.K.',
+                 'raw_affiliations': [
+                     {'value': 'Univ. of Pennsylvania, Philadelphia, PA (United States)',
+                      'source': 'OSTI'}]}]
+    assert parsed_record.authors == expected
+
+
+def test_collaborations(parsed_record):
+    """Test extracting collaborations"""
+    expected = ['']
+    assert parsed_record.collaborations == expected
+
+
+def test_collection(parsed_record):
+    """Test collection setting"""
+    expected = ['Literature']
+    assert parsed_record.collections == expected
+
+
+def test_date_published(parsed_record):
+    """Test extracting published date"""
+    expected = '1973-11-01T00:00:00Z'
+    assert parsed_record.date_published == expected
+
+
+def test_document_type(parsed_record):
+    """Test extracting document type"""
+    expected = None
+    assert parsed_record.document_type == expected
+
+
+def test_dois(parsed_record):
+    """Test extracting DOIs"""
+    expected = [{'doi': '10.1103/PhysRevD.8.3207',
+                 'material': 'publication',
+                 'source': 'OSTI'}]
+    assert parsed_record.dois == expected
+
+
+def test_osti_id(parsed_record):
+    """Test extracting OSTI id"""
+    expected = '1442851'
+    assert parsed_record.osti_id == expected
+
+
+def test_journal_year(parsed_record):
+    """Test extracting journal year"""
+    expected = '1973'
+    assert parsed_record.journal_year == expected
+
+
+def test_journal_title(parsed_record):
+    """Test extracting journal title"""
+    expected = 'Physical Review. D, Particles Fields'
+    assert parsed_record.journal_title == expected
+
+
+def test_journal_issue(parsed_record):
+    """Test extracting journal issue"""
+    expected = '9'
+    assert parsed_record.journal_issue == expected
+
+
+def test_journal_volume(parsed_record):
+    """Test extracting journal volume"""
+    expected = '8'
+    assert parsed_record.journal_volume == expected
+
+
+def test_language(parsed_record):
+    """Test extracting language"""
+    expected = 'English'
```

Comment:
we don't put this in the record if the language is only English. Enforcing it would be a nice addition to the builder.

Reply:
right. I think the builder should know what defaults it wants to skip, not the crawler. here I just test that language is properly extracted from the json. 

Since language is always English for the 10k OSTI documents I have tested with, I don't actually set it in the builder.

Reply:
so maybe it's a unnecessary property of the parser and an unnecessary test. on the other hand it allows for easy future use